File: src/main/java/instance/common/DiskInit.java
package instance.common;

import cloud.common.NodeConfiguration;
import se.sics.kompics.Init;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-03-18
 *
 */

public class DiskInit extends Init {

	private NodeConfiguration nodeConfiguration;

	public DiskInit(NodeConfiguration nodeConfiguration) {
		this.nodeConfiguration = nodeConfiguration;
	}

	public NodeConfiguration getNodeConfiguration() {
		return nodeConfiguration;
	}

}

File: src/main/java/instance/common/DiskInit.scala
package instance.common

import se.sics.kompics.Init
import cloud.common.NodeConfiguration

/**
 * @author Amir Moulavi
 * @dater 2011-07-16
 *
 */
class DiskInit(nodeConfiguration:NodeConfiguration) extends Init {

  def getNodeConfiguration = nodeConfiguration

}